ivre/db/__init__.py

```diff
@@ -2406,78 +2406,108 @@ class DBFlow(DB):
 class MetaDB(object):
 
     db_types = {
-        "nmap": {},
-        "passive": {},
-        "data": {},
-        "agent": {},
-        "flow": {},
-        "view": {},
+        "nmap": {"http": ("http", "HttpDBNmap"),
```

Comment:
Maybe it would be a good idea to add a few comments to explain the structure of this dictionary.